docker/install.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+## One off install script to setup server on Ubuntu 20.04
+
+cd ~
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt install nginx
+
+cp ~/dev-ops/nginx/sites-available/default /etc/nginx/sites-available/default
+
+sudo systemctl restart nginx
+
+sudo apt install certbot python3-certbot-nginx
+sudo certbot --nginx -d ec.dltx.io
+
+# Add rules to prevent docker from punching a hole through the firewall
+cp ~/dev-ops/ufw/after.rules /etc/ufw/after.rules
+
+# Firewall rules
+sudo ufw allow 22
+sudo ufw allow from 103.1.229.2 to any port 443
```

Comment:
For out template should be open to the world

```suggestion
sudo ufw allow 443
```